imports/plugins/included/marketplace/server/methods.js

```diff
@@ -13,8 +13,8 @@ Meteor.methods({
    * @returns {Object|null} - The user hash if found, null otherwise
```

Comment:
@prosf - This doesn't look like it will ever return null if there are any accounts present